CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added `SAPRFC` source to the library.
 - Added `S3` source to the library.
 - Added `RedshiftSpectrum` source to the library.
+- Added `upload(), download()` method to `S3` source.
```

Comment:
2 typos: `upload()` and `download()` methods*

Reply:
Updated in:

- https://github.com/dyvenia/viadot/pull/603/commits/9c597385a727078ed1cf9771fee80a17d4139435